README.md

```diff
@@ -6,6 +6,7 @@ Download sprite.js and upload it to your project or copy paste the code in a new
 Add the script to an Entity with a Script Component. You will see the following script attributes:
 
 - **textureAsset**: This is the sprite that you want to render. If you want transparency for your sprite make sure that this is a .png file.
+- **eventsEnabled**: This is the toggle to allow events when interacting with the sprite.
```

Comment:
Does this need to be a script attribute? There's probably no need to see this property in the Editor I think. Also there is a certain overhead with having script attributes vs just normal class properties. When eventsEnabled is set it will trigger events which are not really needed in this case. So I think we could get away with having it just as a normal class attribute.